vision/google/cloud/proto/vision/v1/text_annotation_pb2.py

```diff
@@ -558,13 +558,14 @@
   DESCRIPTOR = _TEXTANNOTATION,
   __module__ = 'google.cloud.proto.vision.v1.text_annotation_pb2'
   ,
-  __doc__ = """TextAnnotation contains a structured representation of OCR extracted
-  text. The hierarchy of an OCR extracted text structure is like this:
-  TextAnnotation -> Page -> Block -> Paragraph -> Word -> Symbol Each
+  __doc__ = """``TextAnnotation`` contains a structured representation of OCR extracted text.
```

Comment:
@jonparrott The issues here were all variations on

> Inline interpreted text or phrase reference start-string without end-string.

I'm pretty sure it's because `sphinx-docstring-typing` picked up class names and tried to make them pretty?